swift/test/Ice/location/AllTests.swift

```diff
@@ -268,7 +269,7 @@ func allTests(_ helper: TestHelper) async throws {
 
     try await withThrowingTaskGroup(of: Void.self) { [hello] taskGroup in
         for _ in 0..<1000 {
-            taskGroup.addTask {
+            taskGroup.addTask { @Sendable in
```

Comment:
Not clear why you need this `@Sendable`.

Reply:
I could also fix by making a copy of test, but I think making the closure sendable is simpler here.

Reply:
Making TestHelper sendable doesn't fix it?